### ISSUE TITLE
Use gpg instead of gpg2

### ISSUE THREAD

**Issue by AsciiWolf** [micahflee/torbrowser-launcher#529]
Now that #412 was fixed, please also consider using "gpg" instead of "gpg2". On most Linux distributions nowadays, gpg2 is just a symlink to gpg and on some of them, for example Debian-based systems, the symlink does not exist out-of-box and so using gpg2 adds unnecessary dependency on gnupg2 which is a transitional package that just provides a symlink to gpg, is considered deprecated and will be dropped in the future. On some systems, for example the freedesktop Flatpak runtime, gpg2 is already not available at all.

/cc @micahflee

**Comment by JabbaDesilijicTiure** [comment]
```
terrorist@localhost:~$ gpg --version
gpg (GnuPG) 2.2.20
libgcrypt 1.8.6
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Home: /home/terrorist/.gnupg
Supported algorithms:
Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
Cipher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
        CAMELLIA128, CAMELLIA192, CAMELLIA256
Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
Compression: Uncompressed, ZIP, ZLIB, BZIP2
```

@AsciiWolf :+1:

**Comment by rogers0** [comment]
@AsciiWolf this should be fixed in v0.3.4 already.
